### DISCUSSION TITLE
Welcome to PowerShell Universal discussions!

### DISCUSSION THREAD

**Discussion by adamdriscoll** [ironmansoftware/powershell-universal#3167] [Announcements]
We have enabled discussions to allow for communication on issues within this repository. It's not always necessary to open an issue when a discussion could suffice. We've specifically enabled this for the v5 PowerShell Universal beta period.

**Comment by MikeShepard** [comment]
Tried the v5 beta last week and was very impressed.  Is there an estimated date for its release?  No pressure, just wondering. :-)

**Reply by adamdriscoll** [reply]
We are targeting Q3 to give us time to ensure the quality is there but don't have a firm date yet. It could be sooner if we feel it's ready.

**Comment by Omzig** [comment]
FYI, you have to skip v5.1 and jump directly to v5.2

**Comment by bfwm** [comment]
I'm just getting started with PSU. So I'm starting with the v5 beta. I'm finding that some of the documentation is out of sync. 

An example, I'm looking at the Form Authentication code. There is no code to accept, admin and any password, however, it works by default. 

I need to setup AD based auth, but I'm not sure I can trust provided example code. 

Please advise. 

Also, I couldn't decide if this post belonged in the discussion or issue category. Let me know if I chose incorrectly.

**Reply by Omzig** [reply]
ya, since v5 is still wip, the doc will be quite out of date

**Reply by bfwm** [reply]
I understand. Has the Form Authentication code been moved for local accounts?

**Comment by RockoTheeHut** [comment]
I installed v5 some weeks ago.. are the nightlies current or is it the release 2 build from when it was announced? I guess I’m asking if it is worth updating more frequently? The update notes don’t change so it isn’t clear.

**Reply by adamdriscoll** [reply]
We haven't been great about updating the release notes for v5. I'll make sure we focus on that more. The nightlies are receiving a lot of changes. We were planning on beta3 in may but I think that will happen sooner. We are also planning on publishing nightly integration test results publicly in the near term so you will some idea of the stability of a nightly build.

**Comment by alexrgreenwood** [comment]
As part of my kicking the tyres of the v5 beta im trying to create a script to grab the latest beta , install it and then create a workload using scripts + schedules (my use case is about using PSU as a scheduling engine to sync data and call cloud apis)

the current status of beta 4 around schedules seems to be completely broken , e.g. cant do 

`New-PSUSchedule -Cron "*/2 * * * *" -Script "test.ps1" -Name "test"`
Call failed with status code 400 (Bad Request): POST http://192.168.1.111:5001/api/v1/schedule

admin console says
`Object reference not set to an instance of an object. (Computer: PSU1)`

Its not clear if the beta is out there for these kind of issues to be uncovered or more to give people a preview of the new UI and Blazor capabilites

would it be possible to have "known issues" list on the release notes? 
and should i put in issues if I get errors or post them here ?

**Comment by RockoTheeHut** [comment]
Just curious, I see the tables exist for the optional ps1 config files in the DB, is it enabled? Don’t see them being used. If not enabled, is there a timeframe for when they might be available?

**Reply by adamdriscoll** [reply]
This isn't being used yet. We are now using SQLite for all our in memory configuration data rather than a home rolled memory storage implementation. Because SQLite uses the same Entity Framework layer as the other persistence types, it's giving us the opportunity to iron out the DB queries and build out the schema for storing all this data in a persistent storage location. 

Once we have that, we can then just save all the configuration data in the database and wouldn't necessarily need to have git involved for multi-node installs. You could still use git but it could be a separate syncing process and wouldn't have to run on every node or necessarily even happen in the repository directory. This would really simplify configuration because there are just fewer moving parts. 

I would say it's likely in 5.1.

**Comment by kp-bit** [comment]
Just installed v5 beta on a test server, it looks really good! I tried dumping our existing v4 app/dashboard in there (with 100+ sub pages) and much to my surprise it worked. Is that expected behavior? I was under the impression that we would have to rewrite/migrate existing content. But this will save me a lot of work!

**Reply by adamdriscoll** [reply]
That's just as planned! We actually made few breaking changes to the app framework in v5 so they should mostly just work without any changes.
